internal/helper/converter/help/swagger_model_hierarchy_example.go

```diff
@@ -0,0 +1,66 @@
+//go:build ignore
+// +build ignore
+
+package help
+
+/*
```

Comment:
Is this an example usage? In that instead of adding the commented code can we include in a Readme in the converter directory?

Reply:
Yeah I will put it in a readme after I finish with the data protection, I included it in the PR because I thought you might need it to understand the logics.

Reply:
Got it thanks